platform/linux/src/headless_display_egl.cpp

```diff
@@ -25,6 +26,15 @@ HeadlessDisplay::Impl::Impl() {
         throw std::runtime_error("eglInitialize() failed.\n");
     }
 
+    if (major >= 1 && minor >= 2) {
+        EGLenum api = minor >= 4 ? EGL_OPENGL_API : EGL_OPENGL_ES_API;
```

Comment:
also problematic works for `1.4`, but not for `2.0`? If it works, we /always/ want the OpenGL ES API, since that's what we're mainly targeting. We just happen to be largely compatible with Desktop OpenGL due to the limited featureset we're using.